boot/freeldr/freeldr/settings.c

```diff
@@ -126,11 +126,30 @@ LoadSettings(
         return;
     }
 
-    /* Open the [FreeLoader] section and load the settings */
-    if ((BootMgrInfo.FrLdrSection == 0) &&
-        !IniOpenSection("FreeLoader", &BootMgrInfo.FrLdrSection))
+    BOOLEAN FoundLoaderSection = FALSE;
+    PCSTR LoaderSections[] = {
+        "FreeLoader",
+        "Boot Loader",
+        "FlexBoot",
+        "MultiBoot",
+        NULL,
+    };
+
+    /* Open the first section that matches one of the LoaderSections and load the settings */
```

Comment:
That is not quite what is happening here: If the file contains 2 of those sections, it will not open the first one of them, but the one that appears earlier in the list above. I assume that behaviour is desired, so consider reformulating the comment to match the actual behavior.

Reply:
so it's the last section?

Reply:
A updated my PR see if it's looks good for you